### PR TITLE
authentication WIP

### PR DESCRIPTION
**Pull request by albertngo1**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,8 +1,22 @@
-const express = require('express');
+'use strict';
+
+const app = require('express')();
+const passport = require('passport');
+const FacebookStrategy = require('passport-facebook').Strategy;
+const keys = require('../config/keys.js');
+const pg = require('pg');
```
Comment (Julian-Jurai) [inline server/server.js]: what is pg?

File: server/server.js [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
```
Comment (Julian-Jurai) [inline server/server.js]: what is pg?

File: server/server.js [hunk 3]
```diff
@@ -1,8 +1,22 @@
-const express = require('express');
+'use strict';
+
+const app = require('express')();
+const passport = require('passport');
+const FacebookStrategy = require('passport-facebook').Strategy;
+const keys = require('../config/keys.js');
```
Comment (Julian-Jurai) [inline server/server.js]: no need to import `keys` just use `process.env.KEY_NAME`. Also add your api keys to the `.env` file (create one in the root)

File: server/server.js [hunk 4]
```diff
@@ -1,12 +1,33 @@
...
+const app = require('express')();
+const passport = require('passport');
+const FacebookStrategy = require('passport-facebook').Strategy;
+const pg = require('pg');
 
-const app = express();
 const port = process.env.PORT || 5000;
 
-app.get('/api/hello', (req, res) => {
+passport.use(new FacebookStrategy({
```
Comment (sksea) [inline server/server.js]: We should probably extract services like this into separate files/directories (so we have separation of concerns).

Comment (albertngo1) [reply server/server.js]: agreed, refactoring will go at the end

**Pull request merged**